modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/display/context/AssetPublisherDisplayContext.java

```diff
@@ -1263,6 +1263,9 @@ public List<DropdownItem> getScopeDropdownItems(PortletURL addScopeURL)
 
 		dropdownItemList.add(
 			dropdownItem -> {
+				Group group = _assetPublisherHelper.getItemSelectorScopeGroup(
+					layout.getGroup());
+
```

Comment:
If this is used only here, why add the method to the API? Unless there's some other usages (or an established pattern for this), I think it is better to keep it in the display context as a private method.

Reply:
The use case is not really testable trough playwright, and as we use multiple services in the solution I would much rather prefer an integration test over a unit test.

Reply:
Nevermind. I see the interface is marked as provider type, so it is not a major change. As it simplifies testing, go ahead and send it.